src/scalap/scala/tools/scalap/scalax/rules/Rules.scala

```diff
@@ -129,7 +127,7 @@ trait StateRules {
   /** Create a rule that succeeds with a list of all the provided rules that succeed.
       @param rules the rules to apply in sequence.
   */
-  def anyOf[A, X](rules: Seq[Rule[A, X]]) = allOf(rules.map(_ ?)) ^^ { opts => opts.flatMap(x => x) }
+  def anyOf[A, X](rules: Seq[Rule[A, X]]) = allOf(rules.map(_.?)) ^^ { opts => opts.flatMap(x => x) }
```

Comment:
It would seem fitting that scalap, as the least-loved part of this repo (now that the ant bits are dearly departed), be allowed to keep the lingering vestiges of its heyday, but all things must end.

I fear the hate-mail I'll get from scalap's five remaining users.